packages/flutter/test/widget/gesture_detector_test.dart

```diff
@@ -64,10 +64,10 @@ void main() {
     Point upLocation = new Point(10.0, 20.0);
 
     Widget widget = new GestureDetector(
-      onVerticalDragUpdate: (double delta) { dragDistance += delta; },
-      onVerticalDragEnd: (Velocity velocity) { gestureCount += 1; },
+      onVerticalDragUpdate: (DragUpdateDetails details) { dragDistance += details.primaryDelta; },
+      onVerticalDragEnd: (DragEndDetails details) { gestureCount += 1; },
       onHorizontalDragUpdate: (_) { fail("gesture should not match"); },
-      onHorizontalDragEnd: (Velocity velocity) { fail("gesture should not match"); },
+      onHorizontalDragEnd: (_) { fail("gesture should not match"); },
```

Comment:
optional nit: I prefer when we always give types because it helps people copying and pasting the code.